common/image_f32.c:155:17: error: 'HUGE' undeclared

I'm trying to run the tests for this project. After running:

`[user go-apriltag]$ CGO_CFLAGS_ALLOW="-fno-strict-overflow" go test`

I get the error message:

```
# github.com/twitchyliquid64/go-apriltag
In file included from image_f32.c:1:
common/image_f32.c: In function 'image_f32_normalize':
common/image_f32.c:155:17: error: 'HUGE' undeclared (first use in this function)
  155 |     float min = HUGE, max = -HUGE;
      |                 ^~~~
common/image_f32.c:155:17: note: each undeclared identifier is reported only once for each function it appears in
FAIL    github.com/twitchyliquid64/go-apriltag [build failed]
```

It would appear that the variable HUGE is not defined anywhere (at least that I could find). Any solution to this?

--- a/apriltag.go
+++ b/apriltag.go
@@ -1,6 +1,6 @@
 package apriltag
 
-// #cgo CFLAGS: -std=gnu99 -fPIC -I. -O4 -fno-strict-overflow
+// #cgo CFLAGS: -std=gnu99 -fPIC -I. -O4
 // #cgo LDFLAGS: -lm
 // #include "apriltag.h"
 // #include "tag36h11.h"

--- a/common/image_f32.c
+++ b/common/image_f32.c
@@ -152,7 +152,7 @@ void image_f32_gaussian_blur(image_f32_t *im, double sigma, int ksz)
 // remap all values to [0, 1]
 void image_f32_normalize(image_f32_t *im)
 {
-    float min = HUGE, max = -HUGE;
+    float min = HUGE_VAL, max = -HUGE_VAL;
 
     for (int y = 0; y < im->height; y++) {
         for (int x = 0; x < im->width; x++) {

--- a/go.mod
+++ b/go.mod
@@ -1,0 +1,3 @@
+module github.com/koeng101/go-apriltag
+
+go 1.17

